tests/Form/Type/SecurityRolesTypeTest.php

```diff
@@ -94,25 +98,6 @@ public function testSubmitWithHiddenRoleData(): void
         $this->assertContains('ROLE_SUPER_ADMIN', $form->getData());
     }
 
-    public function testChoicesAsValues(): void
```

Comment:
Why is this removed?

Reply:
Because this is tested already on the test above, but conditionally only if there are choices_as_values option